python/tests/tsdf_tests.py

```diff
@@ -18,6 +18,8 @@ def setUp(self):
                       .config("spark.jars.packages", "io.delta:delta-core_2.12:0.7.0") \
                       .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
                       .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
+                      .config("spark.driver.extraJavaOptions", "-Dio.netty.tryReflectionSetAccessible=true") \
+                      .config("spark.executor.extraJavaOptions", "-Dio.netty.tryReflectionSetAccessible=true") \
```

Comment:
Added to mitigate issue identified in [SPARK-29923](https://github.com/apache/spark/pull/26552)